pipelines/ONT/Preprocessing/ONTFlowcell.wdl

```diff
@@ -51,13 +51,11 @@ workflow ONTFlowcell {
 
     String outdir = sub(gcs_out_root_dir, "/$", "") + "/ONTFlowcell/~{dir_prefix}"
 
-    String PU = "unknown"
-    String DT = "2021-01-01T12:00:00.000000-05:00"
     if (defined(final_summary)) {
         call ONT.GetRunInfo { input: final_summary = select_first([final_summary]) }
-        String PU = GetRunInfo.run_info['instrument']
-        String DT = GetRunInfo.run_info['started']
     }
+    String PU = if defined(final_summary) then GetRunInfo.run_info['instrument'] else "unknown"
```

Comment:
```suggestion
    String PU = select_first([GetRunInfo.run_info['instrument'],  "unknown"])
```

Reply:
womtool fails with the following error when using `select_first([])` instead of `if-else`. 
`Failed to process workflow definition 'ONTFlowcell' (reason 1 of 1): Failed to process declaration 'String PU = select_first([GetRunInfo.run_info["instrument"], "unknown"])' (reason 1 of 1): Failed to process expression 'select_first([GetRunInfo.run_info["instrument"], "unknown"])' (reason 1 of 1): Type evaluation failed for IndexAccess(IdentifierMemberAccess(GetRunInfo,run_info,Vector()),StringLiteral(instrument)). Cannot dereference a Map[String, String]? value using a String ke`y
Will need to revert it back